pkg/kubelet/kubeletconfig/controller.go

```diff
@@ -28,6 +28,8 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig"
 	"k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig/validation"
+
```

Comment:
Why empty line here?

Reply:
I wanted to separate the general kubernetes imports from the imports that are specific to the controller (`pkg/kubelet/kubeletconfig`), since there are so many. Makes it a little easier to read.